docs/release-notes/Percona-Everest-1.1.0-(2024-08-02).md

```diff
@@ -0,0 +1,127 @@
+# What's new in Percona Everest 1.1.0
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source cloud native database platform that helps provision and manage databases faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 1.1.0 introduces the following changes:
+
+
+## Release highlights
+
+!!! warning
+    Percona Everest 1.1.0 comes with certain [limitations](Percona-Everest-1.1.0-(2024-08-02).md#known-limitations) that you need to take into account before you update to this version.
+
+### Enhancements in the functionality for PostgreSQL backups and schedules
+
+We're thrilled to announce that we have enhanced our on-demand backups and schedules to address certain limitations with PostgreSQL. These improvements are designed to offer our users a more seamless experience when utilizing these features.
+
+Check out these improvements:
```

Comment:
This doesn't sound like much of an improvement... :sweat_smile: 

Instead, we should probably mention something like we improved the robustness of the backups/restores by enforcing a few limitations in the management of the backup storages, thus preventing some bugs from being triggered in edge case scenarios.

Then we can proceed by listing the limitations we added.

Reply:
Done